arbnode/inbox_tracker.go

```diff
@@ -302,7 +302,7 @@ func (t *InboxTracker) PopulateFeedBacklog(broadcastServer *broadcaster.Broadcas
 		if err != nil {
 			return fmt.Errorf("error getting message %v: %w", seqNum, err)
 		}
-		feedMessage, err := broadcastServer.NewBroadcastFeedMessage(*message, seqNum)
+		feedMessage, err := broadcastServer.NewBroadcastFeedMessage(*message, seqNum, nil)
```

Comment:
Try getting each result with txStreamer.ResultAtCount().
Each result you could get - add to the BroadcastMessage. If there was no result or some other error - add message with the nil result.

Reply:
Done.